### ISSUE TITLE
Crash in klusrmgr when WarpIN not installed

### ISSUE THREAD

**Issue by altsan** [bitwiseworks/rpm-issues#16]
**Describe the bug**

When updating libc on a barebones install without WarpIN installed, the dependency triggers installation of klusrmgr which then executes for initialization. However, the lack of WarpIN results in the following error:

```
Line 17 of _SMBUsersCheck in klusrmgr.VRM:
+++               WarpINPath = left(WarpINPath, length(WarpINPath) - 1 )
*** Error 40 Incorrect call to routine
```
This is because WarpINPath was retrieved using VRGetIni(), which returns an empty string on failure, and thus the expression length(WarpINPath)-1 is invalid.

This crash results in a VX-REXX error popup, which is problematic because this install operation is taking place in an unattended environment and the popup causes the process to hang indefinitely.

**To Reproduce**
1. Install klusrmgr via yum on a system without WarpIN installed.

**Expected behavior**
klusrmgr should either initialize successfully without any error popups if WarpIN is absent, or else fail silently.

**Comment by SilvanScherrer** [comment]
this is a clear regression, which should not be there. We work on a fix asap.

**Comment by herwigb** [comment]
Fixed in klusrmgr 1.4.4
